cps-transformation.lisp

```diff
@@ -573,9 +609,39 @@
 ;; continuation unless absolutely necessary.
 (defmethod tx-cps ((elm if-statement) statement-tail)
 
-  (if (null statement-tail)
-    ;; No special handling is required when there's no statement tail
-    (call-next-method)
+  (cond
```

Comment:
This introduces some logic to handle labelled if-blocks by wrapping them in labelled statement blocks, but as it turns out, the `then` and `else` blocks can be labelled as well, and those can't be handled faithfully by wrapping the whole `if`, because of break-label visibility.
So this is incomplete without https://github.com/chumsley/jwacs/pull/3/commits/326635d01ea6e15b138a05a943c36a1a3572baa2, but that's a rewrite of `if`, so I left it as a separate commit ..